docs/pages/kubernetes-access/getting-started/cluster.mdx

```diff
@@ -23,18 +29,29 @@ Let's deploy Teleport in a Kubernetes with SSO and Audit logs:
 />
 
 ## Prerequisites
+- A registered domain name. This is required for Teleport to set up TLS via Let's Encrypt and for Teleport clients to verify the Proxy Service host.
+
+- A Kubernetes cluster hosted by a cloud provider, which is required for the load balancer we deploy in this guide.
+
+<Admonition title="Local-only setups" type="tip">
+Teleport also supports Kubernetes in on-premise and air-gapped environments. If you would like to try out Teleport on your local machine, we recommend following our [Docker Compose guide](../../getting-started/docker-compose.mdx). 
+
+You can also [contact us](/contact-us/) so we can work with you to determine the best setup for your Kubernetes environment.
+</Admonition>
 
 (!docs/pages/includes/kubernetes-access/helm-k8s.mdx!)
 
 (!docs/pages/includes/permission-warning.mdx!)
 
 ## Step 1/3. Install Teleport
 
-Let's start with a single-pod Teleport using persistent volume as a backend.
+Let's start with a single-pod Teleport using persistent volume as a backend. Modify the values of `CLUSTER_NAME` and `EMAIL` according to your environment, where `CLUSTER_NAME` is the domain name you are using for your Teleport deployment and `EMAIL` is an email address used for notifications.
 
 <Tabs>
   <TabItem label="Open Source">
     ```code
+    $ CLUSTER_NAME="tele.example.com"
```

Comment:
```suggestion
    $ export CLUSTER_NAME="tele.example.com"
```

Export is required in order for `CLUSTER_NAME` to remain set for future commands.

Reply:
Since `CLUSTER_NAME` and `EMAIL` are only used in the initial shell environments of the two code snippets, and their values are passed to the `helm install` command via parameter expansion (i.e. the variables themselves aren't part of the environment of `helm install`), would it be safer to leave these variables unexported?